### PR TITLE
Update tests for indirectbr change

### PR DESCRIPTION
**Pull request by dschuff**

### CODE REVIEW

File: src/test/emwasm_compile_known_gcc_test_failures.txt [hunk 1]
```diff
@@ -51,13 +45,10 @@
 
 
 # Computed gotos (but works with bare wasm clang)
-	20041214-1.c O0
-	20071220-1.c
+	20071220-1.c O3
 	20071220-2.c
```
Comment (aheejin) [inline src/test/emwasm_compile_known_gcc_test_failures.txt]: Why do we still have computed gotos?

File: src/test/emwasm_compile_known_gcc_test_failures.txt [hunk 2]
```diff
@@ -51,13 +45,10 @@
...
 	990208-1.c # no computed goto, but a blockaddress instruction that we can't select
 
-# Untriaged
-	pr56982.c O3
```
Comment (aheejin) [inline src/test/emwasm_compile_known_gcc_test_failures.txt]: This is not a computed goto thing but setjmp fix: #550

File: src/test/emwasm_compile_known_gcc_test_failures.txt (hunk 1 above)
Comment (dschuff) [reply src/test/emwasm_compile_known_gcc_test_failures.txt]: Turns out if you do a blockaddress without an indirectbr the indirectbr lowering pass doesn't lower it away, which makes sense. That still can't really be supported in wasm (e.g. there is no address per se, so no use of the block address other than as a branch target makes sense).

File: src/test/emwasm_compile_known_gcc_test_failures.txt [hunk 3]
```diff
@@ -5,33 +5,30 @@
...
 	20030501-1.c
-	20040302-1.c
 	20040308-1.c
 	20040423-1.c
 	20040520-1.c
 	20041218-2.c
 	20061220-1.c
 	20070919-1.c
-	20071210-1.c
+  20071220-1.c O3
```
Comment (aheejin) [inline src/test/emwasm_compile_known_gcc_test_failures.txt]: Oh no I tried to fix the indentation errors by 'resolve conflicts' tool, which didn't show me tabs, and now they are even more unaligned no

**Pull request merged**